skimage/filters/thresholding.py

```diff
@@ -599,7 +616,7 @@ def threshold_li(image, *, tolerance=None):
     return threshold
 
 
-def threshold_minimum(image, nbins=256, max_iter=10000):
+def threshold_minimum(image, nbins=256, source_range='image', max_iter=10000):
```

Comment:
This is an API change and has to go at the end of the function, preferable with a keyword-only flag to prevent future similar problems